src/pages/Home/components/Card/Card.tsx

```diff
@@ -0,0 +1,28 @@
+import { useNavigate } from "react-router-dom";
+import styles from "./styles.module.scss";
+
+type Props = {
+    title: string;
+    icon: string;
+    color: string;
+    to: string;
+};
+
+const Card = ({ title, icon, color, to }: Props) => {
+    const navigate = useNavigate();
+    return (
+        <div
+            className={`${styles.card}`}
+            style={{ backgroundColor: color }}
```

Comment:
Same suggestion as before, try avoiding inline styles and handle it with classnames and css.